src/main/java/betterquesting/api2/client/gui/controls/PanelTextField.java

```diff
@@ -805,7 +812,18 @@ public boolean onMouseClick(int mx, int my, int button)
                 this.isFocused = true;
                 updateScrollBounds(); // Just in case
             }
-            
+
+            if(clearOnRightClick)
```

Comment:
Should this also be checking the value of `button`, for if it's a right- or left-click?

Reply:
Oops, yes you are correct.